### ISSUE TITLE
Очистилась раскраска строк после сохранения сценария

### ISSUE THREAD

**Issue by ZhokhovM** [Pr-Mex/vanessa-automation#1457]
Укажите:
Версия Vanessa Automation 1.2.037.12
Версия платформы 8.3.19.1264
Режим совместимости базы, где запускается Vanessa Automation Такси
Операционная система Windows Server 2016 Standard

> Заполните шаблон обсуждения GIT

Сценарий:
1.Выполнить любой сценарий в обработке VA.
2.Дождаться выполнения сценария. В сценарии есть раскраска строк.
3.Нажать "Сохранить" или "Сохранить как".
Причина: Очищает раскраску строк.
Ожидалось увидеть раскраску строк без повторного запуска сценария.

**Comment by Pr-Mex** [comment]
@ZhokhovM 
Да, можно добавить флаг: Очищать раскраску при сохранении сценария.
